tests/dataconverter/test_validation.py

```diff
@@ -867,24 +867,114 @@ def format_error_message(msg: str) -> str:
                 TEMPLATE, "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type", "Wrong option"
             ),
             [
-                "The value at /ENTRY[my_entry]/NXODD_name[nxodd_name]/type should "
-                "be one of the following"
-                ": ['1st type', '2nd type', '3rd type', '4th type']."
+                "The value 'Wrong option' at /ENTRY[my_entry]/NXODD_name[nxodd_name]/type "
+                "should be one of the following: "
+                "['1st type', '2nd type', '3rd type', '4th type']."
             ],
             id="wrong-enum-choice",
         ),
         pytest.param(
             alter_dict(
-                TEMPLATE,
-                "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type2",
-                "a very different type",
+                alter_dict(
+                    alter_dict(
+                        alter_dict(
+                            TEMPLATE,
+                            "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type2",
+                            "a very different type",
+                        ),
+                        "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type2/@custom",
+                        True,
+                    ),
+                    "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type2/@attribute_with_open_enum",
+                    "3rd option",
+                ),
+                "/ENTRY[my_entry]/NXODD_name[nxodd_name]/type2/@attribute_with_open_enum_custom",
+                True,
```

Comment:
`attribute_with_open_enum_` it should be an attribute name as described in PR description, in this case `3rd option`?